src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -10472,6 +10472,1942 @@ public class Class
+        [InlineData("/// <summary></summary>")]
+        public void ExtraCloseCurly_LeadingTriviaCases(string closeCurlyLeadingTrivia)
+        {
+            // Test all the different type-only member forms.
+            var text = $$"""
+                namespace N
+                {
+                    class Type
+                    {
+                        {{closeCurlyLeadingTrivia}}
+                        }
+
+                        private Constructor() { }
+                        {{(closeCurlyLeadingTrivia.Contains("#") ? "#endif" : "")}}
+                    }
+                }
+                """;
+            UsingTree(text, options: CSharpParseOptions.Default.WithDocumentationMode(DocumentationMode.Parse),
+                // (6,9): error CS1519: Invalid token '}' in class, record, struct, or interface member declaration
+                //         }
+                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "}").WithArguments("}").WithLocation(6, 9));
+
+            N(SyntaxKind.CompilationUnit);
+            {
+                N(SyntaxKind.NamespaceDeclaration);
+                {
+                    N(SyntaxKind.NamespaceKeyword);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "N");
+                    }
+                    N(SyntaxKind.OpenBraceToken);
+                    N(SyntaxKind.ClassDeclaration);
+                    {
+                        N(SyntaxKind.ClassKeyword);
+                        N(SyntaxKind.IdentifierToken, "Type");
+                        N(SyntaxKind.OpenBraceToken);
+                        N(SyntaxKind.ConstructorDeclaration);
+                        {
+                            N(SyntaxKind.PrivateKeyword);
+                            N(SyntaxKind.IdentifierToken, "Constructor");
+                            N(SyntaxKind.ParameterList);
+                            {
+                                N(SyntaxKind.OpenParenToken);
+                                N(SyntaxKind.CloseParenToken);
+                            }
+                            N(SyntaxKind.Block);
+                            {
+                                N(SyntaxKind.OpenBraceToken);
+                                N(SyntaxKind.CloseBraceToken);
+                            }
+                        }
+                        N(SyntaxKind.CloseBraceToken);
+                    }
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.EndOfFileToken);
+            }
+            EOF();
+        }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74482")]
+        public void ExtraCloseCurly_WithSemicolon()
+        {
+            var text = $$"""
+                namespace N
+                {
+                    class Type
+                    {
+                        };
+
+                        // Will not move into type
```

Comment:
basically, it would mean having to remove that semicolon as well and make into skipped trivia.  i basically handled the common/normal/easy case.  and this test just demonstrates that (and will catch changes here).